src/textual/parser.rs

```diff
@@ -276,72 +280,79 @@ impl<'a> Parser<'a> {
 }
 
 #[cfg(test)]
-fn query_roundtrip_test(input: &str) {
-    let mut nu = SymbolStore::new();
-    let mut p = Parser::new(&mut nu);
-
-    let q = p.parse_query_str(input).unwrap();
+mod test {
```

Comment:
Nit: now that you moved all of these test functions into a `test` namespace, could you remove the `test_` prefix from them? Reads a bit nicer in the test output to not have `test::test_` there.

Reply:
Done.